File: Python/1013.py
a, b, c = input().split()

a = int(a)
b = int(b)
c = int(c)

print('{:.0f} eh o maior'.format(((((a + b + abs(a-b))/2) + c + abs(((a + b + abs(a-b))/2) - c))/2)))

File: Ruby/1013.rb
a, b, c = gets.split
a = a.to_i
b = b.to_i
c = c.to_i

if a >= b and a >= c
    puts(a.to_s + " eh o maior")
elsif b >= a and c >= a
    puts(b.to_s + " eh o maior")
elsif c >= a and c >= b
    puts(c.to_s + " eh o maior")
end